specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/examples/SqlPoolVulnerabilityAssessmentScanRecordsGet.json

```diff
@@ -21,7 +21,6 @@
           "startTime": "2017-12-12T17:45:06Z",
           "endTime": "2017-12-12T17:47:06Z",
           "errors": [],
-          "storageContainerPath": "https://myaccount.blob.core.windows.net/vulnerability-assessment",
```

Comment:
if it's a write only property do we not need to remove it from other API version examples